src/media/local-microphone-track.ts

```diff
@@ -1,6 +1,6 @@
-import { LocalTrack } from './local-track';
+import { LocalVideoTrack } from './local-video-track';
 
 /**
  * Represents a local track for a microphone source.
  */
-export class LocalMicrophoneTrack extends LocalTrack {}
+export class LocalMicrophoneTrack extends LocalVideoTrack {}
```

Comment:
LocalMicrophoneTrack should extend LocalAudioTrack

Reply:
done